run_dir/static/js/pricing_quote.js

```diff
@@ -580,7 +592,7 @@ app.component('v-pricing-quote', {
               </div>
               <div class="card mt-5">
                 <div class="card-header">
-                  <h4>Preview</h4>
+                  <h4>Preview <small>Using template: {{this.template_text_data.doc_id}}-{{this.template_text_data.edition}}</small></h4>
```

Comment:
Is this supposed to be centered? Maybe no data is being fetched locally, so it looks weird 🤷🏻‍♀️

Reply:
Its up on stage now.